### PR TITLE
Spanish translation for Ecto lesson

### PR DESCRIPTION
**Pull request by milmazz**
This lesson is almost done, I just need to translate the following sections:

- [x] Fragments
- [x] Changesets

/cc @doomspork @erickgnavar

### CODE REVIEW

File: es/lessons/specifics/ecto.md [hunk 1]
```diff
@@ -0,0 +1,344 @@
...
+
+```elixir
+query = from u in User,
+    where: u.confirmed == true,
+    select: count(u.id)
+```
+
+### Group By
+
+Para agrupar el número de nombres de usuarios en base a su estado de confirmación de cuenta podemos incluir la opción `group_by`:
```
Comment (jesusangelm) [inline es/lessons/specifics/ecto.md]: Para agrupar nombres de usuarios de acuerdo a su fecha de creación podemos incluir la opción `group_by`:

**Comment by doomspork** [comment]
@milmazz apologizes but I am out of town for a funeral so it won't be until early next week that I can look this over.

Thank you again for your contributions and have a Happy New Year! :+1:

**Comment by milmazz** [comment]
@doomspork My condolences! 

No problem at all, review this when you can, I'm happy to help!

Happy new year for you too! :boom:

**Comment by doomspork** [comment]
@erickgnavar how would you like to go about merging these lessons?  Are you almost finished with the basic ones and the configuration?

**Comment by doomspork** [comment]
ping @erickgnavar :grinning:

**Comment by erickgnavar** [comment]
Hi @doomspork, sorry for the late response, I would like to finish the basic lessons before doing the merge, I hope to finish in 2 days.

**Comment by doomspork** [comment]
Thanks for the update @erickgnavar :+1:

**Comment by doomspork** [comment]
Hi @erickgnavar!  I want to see whether you are be open to merging both this PR and your's (#43) even though the remainder of the Basics are not translated.  My thinking behind it is:

+ The Spanish language setup will be completed.  This unblocks other contributors who might be interested in lending a hand (see @milmazz / @ZuraGuerra)
+ I'd like to merge in @milmazz changes; I don't want PR rot to discourage contributors.

Open to your thoughts and feedback as well as others :grinning:

Thanks again for all your work

**Comment by erickgnavar** [comment]
Sure, no problem, I have 6 lessons translated, you can merge with this PR.
I'm still working on the remaining lessons.

**Comment by doomspork** [comment]
Thanks @erickgnavar.  I left you a comment on your PR RE: rebasing, everything else looks good. 

@milmazz once I merge in #43 I'll merge and deploy this.  Thank you again.  Sorry for the delays.

**Pull request merged**